src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml

```diff
@@ -0,0 +1,220 @@
+                <Track.Thumb>
+                    <Thumb Style="{DynamicResource LibraryVerticalThumb}" />
+                </Track.Thumb>
+            </Track>
+        </Grid>
+    </ControlTemplate>
+
+    <ControlTemplate x:Key="LibraryScrollViewerControlTemplate"
+                     TargetType="{x:Type ScrollViewer}">
+        <Grid x:Name="Grid"
+              Background="{TemplateBinding Background}">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition />
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <ScrollContentPresenter x:Name="PART_ScrollContentPresenter"
+                                    CanContentScroll="{TemplateBinding CanContentScroll}"
+                                    CanHorizontallyScroll="False"
+                                    CanVerticallyScroll="True"
+                                    ContentTemplate="{TemplateBinding ContentTemplate}"
+                                    Content="{TemplateBinding Content}"
+                                    Grid.RowSpan="2"
+                                    Margin="{TemplateBinding Padding}"
+                                    Grid.ColumnSpan="2" />
+            <ScrollBar x:Name="PART_VerticalScrollBar"
+                       AutomationProperties.AutomationId="VerticalScrollBar"
+                       Cursor="Arrow"
+                       Grid.Column="1"
+                       Maximum="{TemplateBinding ScrollableHeight}"
+                       Minimum="0"
+                       Grid.Row="0"
+                       Visibility="{TemplateBinding ComputedVerticalScrollBarVisibility}"
+                       Value="{Binding VerticalOffset, Mode=OneWay, RelativeSource={RelativeSource TemplatedParent}}"
+                       ViewportSize="{TemplateBinding ViewportHeight}"
+                       Width="8"
+                       Style="{DynamicResource LibraryScrollBarStyle}" />
+        </Grid>
+    </ControlTemplate>
+
+    <!--EndRegion-->
+
+    <!-- Region TooltipWindow -->
+
+    <ControlTemplate x:Key="TooltipThumbTemplate"
+                     TargetType="{x:Type Thumb}">
+        <Grid>
+            <Rectangle HorizontalAlignment="Stretch"
+                       VerticalAlignment="Stretch"
+                       Width="Auto"
+                       Height="Auto"
+                       Fill="Transparent" />
+            <Border x:Name="Rectangle1"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Width="5"
+                    Height="Auto"
+                    Background="#aaaaaa"
+                    Opacity="0.5"
+                    CornerRadius="2" />
+        </Grid>
+    </ControlTemplate>
+
+    <ControlTemplate x:Key="TooltipScrollBarTemplate"
```

Comment:
`LibraryScrollBarTemplate` and `TooltipScrollBarTemplate` can be merged, if we can merge the above two.